dts: Simplify the description of the binding

rruuaanng opened the pull request:
As title.

henrikbrixandersen reviewed (dismissed):
Thanks.

kartben commented on dts/bindings/riscv/openisa,rv32m1-pcc.yaml:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: RV32M1 PCC (Peripheral Clock Control) IP node
+description: RV32M1 PCC (Peripheral Clock Control) IP
```
"IP" can also go away afaict

```suggestion
description: RV32M1 PCC (Peripheral Clock Control)
```

kartben commented on dts/bindings/sd/zephyr,mmc-disk.yaml:
```diff
@@ -1,5 +1,5 @@
 description: |
-  Zephyr MMC disk node. A binding with this compatible present within an SD
+  Zephyr MMC disk. A binding with this compatible present within an SD
```
```suggestion
  Zephyr MMC disk.
  
  A binding with this compatible present within an SD
```

kartben commented on dts/bindings/sd/zephyr,sdmmc-disk.yaml:
```diff
@@ -1,5 +1,5 @@
 description: |
-  Zephyr SDMMC disk node. A binding with this compatible present within an SD
+  Zephyr SDMMC disk. A binding with this compatible present within an SD
```
```suggestion
  Zephyr SDMMC disk.
  
  A binding with this compatible present within an SD
```

kartben commented on dts/bindings/shi/ite,it8xxx2-shi.yaml:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: ITE, IT8XXX2 Serial Host Interface (SHI) node
+description: ITE, IT8XXX2 Serial Host Interface (SHI)
```
```suggestion
description: ITE IT8XXX2 Serial Host Interface (SHI)
```

kartben commented on dts/bindings/shi/nuvoton,npcx-shi-enhanced.yaml:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: Nuvoton, NPCX Serial Host Interface (SHI) node
+description: Nuvoton, NPCX Serial Host Interface (SHI)
```
```suggestion
description: Nuvoton NPCX Serial Host Interface (SHI)
```

kartben commented on dts/bindings/shi/nuvoton,npcx-shi.yaml:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: Nuvoton, NPCX Serial Host Interface (SHI) node
+description: Nuvoton, NPCX Serial Host Interface (SHI)
```
```suggestion
description: Nuvoton NPCX Serial Host Interface (SHI)
```

kartben commented on dts/bindings/spi/cypress,psoc6-spi.yaml:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: Cypress SBC[SPI]
+description: Cypress SBC SPI
```
```suggestion
description: Cypress SBC SPI controller
```

kartben commented on dts/bindings/spi/microchip,mpfs-qspi.yaml:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: Microchip Polarfire SOC QSPI IP node
+description: Microchip Polarfire SOC QSPI IP
```
```suggestion
description: Microchip Polarfire SOC QSPI controller
```

kartben commented on dts/bindings/spi/microchip,mpfs-spi.yaml:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: Microchip Polarfire SOC SPI IP node
+description: Microchip Polarfire SOC SPI IP
```
```suggestion
description: Microchip Polarfire SOC SPI controller
```

kartben commented on dts/bindings/spi/snps,designware-spi.yaml:
```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: Synopsys DesignWare SPI node
+description: Synopsys DesignWare SPI
```
```suggestion
description: Synopsys DesignWare SPI controller
```

kartben commented on dts/bindings/spi/ti,cc13xx-cc26xx-spi.yaml:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: TI SimpleLink CC13xx / CC26xx SPI node
+description: TI SimpleLink CC13xx / CC26xx SPI
```
```suggestion
description: TI SimpleLink CC13xx / CC26xx SPI controller
```

kartben commented on dts/bindings/spi/xlnx,xps-spi-2.00.a.yaml:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: Xilinx AXI Quad SPI IP node
+description: Xilinx AXI Quad SPI IP
```
```suggestion
description: Xilinx AXI Quad SPI controller
```

kartben commented on dts/bindings/tach/ene,kb1200-tach.yaml:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: ENE, KB1200-Tachometer node
+description: ENE, KB1200-Tachometer
```
```suggestion
description: ENE KB1200 Tachometer
```

kartben commented on dts/bindings/tach/ite,it8xxx2-tach.yaml:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: ITE, it8xxx2 Tachometer node
+description: ITE, it8xxx2 Tachometer
```
```suggestion
description: ITE IT8xxx2 Tachometer
```

kartben commented on dts/bindings/tach/nuvoton,npcx-tach.yaml:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: Nuvoton, NPCX-Tachometer node
+description: Nuvoton, NPCX-Tachometer
```
```suggestion
description: Nuvoton NPCX Tachometer
```

kartben commented on dts/bindings/timer/nuvoton,npcx-itim-timer.yaml:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: Nuvoton, NPCX Internal Timer (ITIM) node
+description: Nuvoton, NPCX Internal Timer (ITIM)
```
```suggestion
description: Nuvoton NPCX Internal Timer (ITIM)
```

kartben commented on dts/bindings/timer/silabs,gecko-timer.yaml:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: Silabs gecko timer node
+description: Silabs gecko timer
```
```suggestion
description: Silabs Gecko timer
```

bjarki-andreasen reviewed (commented):
Looks good for RTC

rruuaanng committed:
dts: spi: Simplify the description of the binding

Remove redundant descriptions in SPI bindings, such
as "... node".

Signed-off-by: James Roy <rruuaanng@outlook.com>

rruuaanng committed:
dts: tach: Simplify the description of the binding

Remove redundant descriptions in tach bindings, such
as "... node".

Signed-off-by: James Roy <rruuaanng@outlook.com>

rruuaanng committed:
dts: timer: Simplify the description of the binding

Remove redundant descriptions in timer bindings, such
as "... node".

Signed-off-by: James Roy <rruuaanng@outlook.com>

rruuaanng committed:
dts: watchdog: Simplify the description of the binding

Remove redundant descriptions in watchdog bindings, such
as "... node" and " ... driver binding".

Signed-off-by: James Roy <rruuaanng@outlook.com>

rruuaanng committed:
dts: usb: Simplify the description of the binding

Remove redundant descriptions in USB bindings, such
as "... node".

Signed-off-by: James Roy <rruuaanng@outlook.com>

rruuaanng committed:
dts: rtc: Simplify the description of the binding

Remove redundant descriptions in RTC bindings, such
as "... node", "... common bindings" and "This binding provides".

Signed-off-by: James Roy <rruuaanng@outlook.com>

rruuaanng committed:
dts: sd: Simplify the description of the binding

Remove redundant descriptions in SD bindings, such
as "... node".

Signed-off-by: James Roy <rruuaanng@outlook.com>

rruuaanng committed:
dts: riscv: Simplify the description of the binding

Remove redundant descriptions in riscv bindings, such
as "... node".

Signed-off-by: James Roy <rruuaanng@outlook.com>

rruuaanng committed:
dts: shi: Simplify the description of the binding

Remove redundant descriptions in shi bindings, such
as "... node".

Signed-off-by: James Roy <rruuaanng@outlook.com>

rruuaanng committed:
dts: sdhc: Simplify the description of the binding

Remove redundant descriptions in SDHC bindings, such
as "... node".

Signed-off-by: James Roy <rruuaanng@outlook.com>

rruuaanng committed:
dts: rng: Simplify the description of the binding

Remove redundant descriptions in RNG bindings, such
as "... node".

Signed-off-by: James Roy <rruuaanng@outlook.com>

rruuaanng committed:
dts: smbus: Simplify the description of the binding

Remove redundant descriptions in smbus bindings, such
as "... node".

Signed-off-by: James Roy <rruuaanng@outlook.com>

rruuaanng committed:
dts: sram: Simplify the description of the binding

Remove redundant descriptions in SRAM bindings, such
as "... description".

Signed-off-by: James Roy <rruuaanng@outlook.com>
